examples/lorawan/README.md

```diff
@@ -64,7 +65,7 @@ for that device.
 
 1. flash device with appropriate keys and test
 
-    $ DEVEUI=<device eui> APPEUI=<application eui> APPKEY=<application key> make -C examples/lorawan/ flash test
+    $ CONFIG_LORAMAC_DEV_EUI_DEFAULT=<device eui> CONFIG_LORAMAC_APP_EUI_DEFAULT=<application eui> DCONFIG_LORAMAC_APP_KEY_DEFAULT=<application key> make -C examples/lorawan/ flash test
```

Comment:
I think for now we should keep the Make variables to pass the credentials. It still can be overridden by menuconfig and I believe there are some tests that are depending on this.

Reply:
Oh okay. I shall undo it.

Reply:
BTW Should we update the tests?